Misc/NEWS

```diff
@@ -144,6 +144,12 @@ Build
 Tests
 -----
 
+- bpo-30207: The test.test_support module was converted into package and
```

Comment:
I'd suggest starting the NEWS entry with the rationale for the change: "To simplify backports from Python 3, ..."